examples/spark/images/base/core-site.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
+  <property>
+    <name>fs.gs.impl</name>
+    <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
```

Comment:
The conversation piece we discussed yesterday is whether how to pass in the configs.  By default this enables some fs options, when in reality you want the user to explicitly specify the options.  Gluster, Tachyon, Orange, etc.  

A much more general option is to create a pattern for either mounting configs or enabling side-car containers.